charts/dbildungs-iam/templates/dbildungs-iam-deployment-backend.yaml

```diff
@@ -1,22 +1,23 @@
 apiVersion: apps/v1
```

Comment:
Please check the security context settings

Reply:
done